processing/src/main/java/org/apache/druid/segment/UnnestStorageAdapter.java

```diff
@@ -327,8 +327,7 @@ to generate filters to be passed to base cursor (filtersPushedDownToBaseCursor)
               (BooleanFilter) queryFilter,
               inputColumn,
               inputColumnCapabilites,
-              filterSplitter,
-              isTopLevelAndFilter
+              filterSplitter
```

Comment:
We should remove all occurrences of `isTopLevelAndFilter` which is not doing anything